tools/cloud-build/images/cluster-toolkit-dockerfile/README.md

```diff
@@ -0,0 +1,57 @@
+# Cluster Toolkit Dockerfile
+
+This repository contains a Dockerfile for building a Docker image with Cluster Toolkit and its dependencies installed and the `gcluster` binary readily available for use.
+
+## System Requirements
+
+* [Docker Engine](https://docs.docker.com/engine/) needs to be installed to build a Docker image from this Dockerfile.
+
+## Build Arguments
+The following build arguments can be used to customize the build process:
+* **`BASE_IMAGE`**: The base image to use for the build. Defaults to `gcr.io/google.com/cloudsdktool/google-cloud-cli:stable`.
+* **`TERRAFORM_VERSION`**: The version of Terraform to install. Defaults to `1.5.2`.
+* **`PACKER_VERSION`**: The version of Packer to install. Defaults to `1.8.6`.
+* **`GO_VERSION`**: The version of Go to install. Defaults to `1.21.0`.
+* **`CLUSTER_TOOLKIT_REF`**: The [Cluster Toolkit repository's](https://github.com/GoogleCloudPlatform/cluster-toolkit/releases) branch or tag from which to build Cluster Toolkit.  Defaults to the `main` branch.
+
+## Build the Cluster Toolkit Docker Image
+To build the Cluster Toolkit Docker image, navigate to the directory the Dockerfile is present in and run the following command:
+
+```bash
+docker build --build-arg BASE_IMAGE=<base_image> \
+             --build-arg TERRAFORM_VERSION=<terraform_version> \
+             --build-arg PACKER_VERSION=<packer_version> \
+             --build-arg GO_VERSION=<go_version> \
+             --build-arg CLUSTER_TOOLKIT_REF=<cluster_toolkit_ref> \
+             -t <image_name> .
+```
+
+Example:
+
+```bash
+docker build --build-arg CLUSTER_TOOLKIT_REF=v1.40.0 -t my-cluster-toolkit-image .
```

Comment:
I this we should make the example show the tag name(s) we'd recommend, i.e. `gcluster` and `ghpc` (instead of `my-cluster-toolkit-image`).

```suggestion
docker build --build-arg CLUSTER_TOOLKIT_REF=v1.40.0 -t gcluster -t ghpc .
```